### ISSUE TITLE
Request using datasets from NUSMods

### ISSUE THREAD

**Issue by ryanYtan** [nus-cs2103-AY1920S1/forum#104]
My feature requires using module datasets taken from [NUSMod's API](https://api.nusmods.com/v2/). The data is stored in the form of `.json` files, allowing me to use the Jackson API.

Am I allowed to download and locally use such datasets?

**Comment by damithc** [comment]
Yes, that should be fine. Remember to package the data in the jar file, or provide another easy means for the user to get the same data.
